css/logo.css

```diff
@@ -0,0 +1,15 @@
+.logo__image {
```

Comment:
If you don't have a "logo" block in the markup, you shouldn't use "logo__element", it contradicts BEM convention